### ISSUE TITLE
Parole da rimuovere da word_list.txt

### ISSUE THREAD

**Issue by pietroppeter** [pietroppeter/wordle-it#76]
word_list.txt contiene tutte le parole che si possono usare per indovinare. È un file abbastanza sporco in cui possono esserci parole che è meglio se non ci sono.

Questo è un problema minore, il giocatore si accorge del problema solo quando tira ad indovinare e spreca un tentativo con una parola che il gioco dovrebbe rigettare. Però sempre un problema.

In questa issue segno (ed altre persone possono partecipare) le parola che andrebbero rimosse, magari prima o poi lo faccio.

Dopo la rimozione dal file comunque ci sono delle operazioni da fare, non basta rimuoverle solo da lì. Vedi https://pietroppeter.github.io/wordle-it/dict/

**Comment by pietroppeter** [comment]
Esempio di due parole che mi sono state segnalate oggi:
- arcai: non è da rimuovere in quanto passato remoto del verbo arcare che a quanto pare esiste (TIL) https://www.treccani.it/vocabolario/arcare/#
- araci: nome di uno stato del Brasile o di una associazione. Andrebbe rimossa

**Comment by linfamonte** [comment]
fanne e dinne

**Comment by pietroppeter** [comment]
> fanne e dinne

usate poco ma mi risulta essere italiano corretto. Imperativo + ne (=di questa cosa/di queste cose). Es:
- fanne tesoro (di questa esperienza)
- "hai delle parole preferite?" "sì" "dinne tre" (di queste parole)

**Comment by linfamonte** [comment]
No, certo che non dico che siano sbagliate!! Intendo solo che le parole con dei pronomi non vengono mai accettate da nessun gioco di parole in genere. (scarabeo per dirne uno)
Purtroppo adesso non posso dirti nello specifico quali altre parole invece non vengono accettate su "Parole" perché farei confusione magari, al momento mi vengono solo molte parole inglesi di uso comune che non vengono accettate (e pur se inglesismi sono correttissime! Login, sitin, altro)

**Comment by ilcarogna** [comment]
oggi 26 agosto 2025 al n°1330 2/6

è stata accettata, come errata, la parola ILLEA (la soluzione è IELLA). Un lemma che non trovo da nessuna parte!
